core/frontend/src/PerModelCategoryVisibility.ts

```diff
@@ -84,6 +87,12 @@ function compareCategoryOverrides(lhs: PerModelCategoryVisibilityOverride, rhs:
   return 0 === cmp ? compareStrings(lhs.categoryId, rhs.categoryId) : cmp;
 }
 
+// TODO: I know the saved-views package defines an interface with the same exact name... Will this cause problems? Maybe I should define these props as part of the PerModelCategoryVisibility namespace.
```

Comment:
Does it have the same properties?

Reply:
Slightly different, the one in the saved views package is: 

modelId: string;
categoryId: string;
visible: boolean;

Reply:
I ended up putting these props as part of the PerModelCategoryVisibility namespace.